Packs/AlphaVantage/Integrations/AlphaVantage/test_data/stock history get.json

```diff
@@ -0,0 +1,712 @@
+{
```

Comment:
The test data files are not used anywhere, did you plan to add unit tests for these or can be removed?

Reply:
I did not plan to add more unit tests, those files are nice to have example for the API responses.